draft-ietf-rats-corim.md

```diff
@@ -1408,36 +1408,51 @@ Evidence MUST be successfully verified.
 
 ### The Accepted Claims Set
 
-At the end of the Evidence collection process evidence has been converted into
-a format suitable for appraisal. To this end, this document describes an `accepted-claims-set`
-format and the algorithms used to compare it against CoMID Reference Values.
+At the end of the Evidence collection process Evidence has been converted into
+a format suitable for appraisal. To this end, this document describes an
+`accepted-claims-set` format and the algorithms used to compare it against
+CoMID Reference Values.
 
 ~~~ cddl
 {::include cddl/accepted-claims-set.cddl}
 ~~~
 
-Verifiers are not required to use this as their internal state, but for the
+Verifiers are not required to use this as their internal state, for the
 purposes of this document a sample Verifier is discussed which uses this format.
 
-The Accepted Claims Set will be matched against CoMID Reference Values, as per
+The Accepted Claims Set (ACS) contains the actual state of Target Environments (TEs).
+The `evidence-triples` field contains Evidence (from Attesters) and Endorsements
```

Comment:
#158

Reply:
There is some work to be done to fulfil the explanation which will be covered by issue #158